# The header provides a time stamp and a reference frame (frame_id)
Header header
# This provides speed
std_msgs/Float64 speed
